pallets/schemas/src/lib.rs

```diff
@@ -220,17 +239,7 @@ pub mod pallet {
 		) -> DispatchResult {
 			let sender = ensure_signed(origin)?;
 
-			ensure!(
-				model.len() >= T::MinSchemaModelSizeBytes::get() as usize,
-				Error::<T>::LessThanMinSchemaModelBytes
-			);
-			ensure!(
-				model.len() <= Self::get_schema_model_max_bytes() as usize,
-				Error::<T>::ExceedsMaxSchemaModelBytes
-			);
```

Comment:
I don't see these moved into `create_schema_for`. Perhaps I am just missing where they moved to however.

Reply:
Good catch!